themes/digital.gov/src/scss/new/shortcodes/_note.scss

```diff
@@ -1,139 +1,146 @@
 @use "uswds-core" as *;
 
-.note {
-  @include u-measure(6);
-  @include u-margin-bottom(2);
-  @include u-padding-y(2.5);
-  @include u-padding-x(3);
-  @include u-text("light");
-  @include u-radius("sm");
-  @include u-bg("blue-5");
-  @include u-text("blue-70");
-  @include u-border-left("05", "solid", "blue-20");
-
-  h4 {
-    @include u-margin-bottom("105");
-    @include u-margin-top(0);
-    @include u-font("sans", "lg");
-    @include u-text("light");
-
-    i {
-      @include u-margin-right("05");
-    }
-  }
+.dg-note {
+  background-color: color("blue-5");
+  border-left-color: color("blue-20");
+  border-left-style: solid;
+  border-left-width: 5px;
+  color: color("blue-70");
+  font-size: size("ui", "md");
+  font-weight: font-weight("light");
+  padding: units(3);
 
   a {
-    @include u-text("no-underline");
-    @include u-text("blue-70");
-    @include u-border-bottom("1px", "solid", "blue-70");
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    color: color("blue-70");
+    text-decoration: none;
 
     &:hover {
       @include u-text("no-underline");
-      @include u-text("blue-80");
+      color: color("blue-70");
     }
   }
 
   .highlight {
-    @include u-bg("blue-10");
+    background-color: color("blue-10");
   }
 
   strong {
-    @include u-text("medium");
+    font-weight: font-weight("medium");
   }
 
   p:last-child {
-    @include u-margin-bottom(0);
+    margin-bottom: 0;
   }
 
   blockquote {
-    @include u-padding-left(0);
-    @include u-border(0);
+    border: 0;
+    padding-left: 0;
   }
 
   ol,
   ul {
-    @include u-margin-y(0);
-    @include u-padding-left("205");
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-left: 20px;
```

Comment:
Please use closes spacing token for left padding.

Reply:
updated with tokens